bot/bot_commands.py

```
@@ -1,20 +1,16 @@
 import discord
-from discord.ext import commands # Required for commands.Bot type hint if used, and for bot.tree.command
+from discord.ext import commands
 import time
-from email_utils import is_valid_email, send_email as email_util_send_email
+from .email_utils import is_valid_email, EmailService
```

Pylint (3.3.7)

[error] Attempted relative import beyond top-level package

(E0402)